Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +38,67 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message)) return "";
+            if (padding < 0) throw new ArgumentOutOfRangeException($"Padding cannot be negative. (value: \"{padding}\")");
+
+            var main = InitializeMain(message, out var maxLength);
+            BuildMainLines(padding, main, maxLength);
+            var paddingLines = BuildPaddingLines(padding, maxLength);
+            var border = $"+{GenSpaces((padding * 2 + maxLength), '-')}+";
+
+            return $"{BuildFinalMessage(border, paddingLines, main)}{Environment.NewLine}";
+        }
+
+        private static List<string> InitializeMain(string message, out int maxLength)
+        {
+            var main = new List<string>();
```

Comment:
Don't use arrays or lists, etc for string building operations. Use ``StringBuilder``, due to optimization reasons and utility method.

Reply:
Ok